internal/cmd/logout_test.go

```diff
@@ -90,9 +92,9 @@ func TestLogout(t *testing.T) {
 		},
 	}
 
-	t.Run("default", func(t *testing.T) {
+	t.Run("with all", func(t *testing.T) {
```

Comment:
should add test cases for current, server, and clear variants

Reply:
done